.circleci/config.yml

```
@@ -727,20 +727,24 @@ commands:
     steps:
       - run: python .circleci/scripts/wait_for_master.py <<parameters.scheme>>://<<parameters.host>>:<<parameters.port>>
 
-  upload-cloud-watch-to-s3:
+  locate-cloud-watch-logs:
     parameters:
       cluster-id:
         type: string
-      bucket:
+      region:
         type: string
-        default: det-ci-cloud-watch-logs
+        default: us-west-2
     steps:
       - run:
-          name: Upload cloud watch logs S3
+          name: Locate cloud watch logs
```

Going to throw in some nitpicks while we're touching these lines...
```suggestion
          name: Locate CloudWatch logs
```